App.js

```diff
@@ -1,26 +1,32 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState, useEffect } from 'react';
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+import Question from './components/Question';
+import Answer from './components/Answer';
+import { ShakeEventExpo } from './components/ShakeEventExpo';
 
 const App = () => {
+  const [firstView, setFirstView] = useState(true);
+
+  const shakePhone = () => {
+    setFirstView(false);
+  };
+
+  useEffect(() => {
+    ShakeEventExpo.addListener(() => {
+      shakePhone();
+    });
+
+    return () => {
+      ShakeEventExpo.removeListener();
+    };
+  }, []);
+
   return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
+    <>
```

Comment:
You don't need React Fragment over here as you are always going to return only one component - either `<Question />` or `<Answer />`